trade-finance-manager-ui/server/controllers/utilisation-reports/edit-payment/index.ts

```diff
@@ -52,8 +63,19 @@ export const postEditPayment = async (req: PostEditPaymentRequest, res: Response
       return res.redirect(`/utilisation-reports/${reportId}`);
     }
 
+    const checkedCheckboxIds = getEditPaymentsCheckboxIdsFromObjectKeys(req.body);
+    const checkedCheckboxIdsRecord = mapCheckedCheckboxesToRecord(checkedCheckboxIds);
+    const isCheckboxChecked = (checkboxId: string): boolean => Boolean(checkedCheckboxIdsRecord[checkboxId]);
```

Comment:
Could we just create a set of checkedCheckboxIds and then check for inclusion in the set? I think that would be simpler and then we wouldn't need to export and make common the `mappedCheckedCheckboxesToRecord` method?